main/guides/zoe/contract-requirements.md

```diff
@@ -103,6 +110,65 @@ Use `zcf.makeInvitation()` to create the first party's `invitation`:
     );
 ```
 
+The third argument (which is optional and wasn't needed for the first `invitation`) says
+the counterparty has to offer an `amount` matching the first party's `want.Price`,
+and shoateArgs) => {
+  ...
+  // your code here
+  return harden({ creatorFacet, creatorInvitation, publicFacet });
+}
+harden(start);
+export { start };
```

Comment:
We're trying to move away from the 
```js
const fn = (...args) => {
  ...body;
};
harden(fn);
export { fn };
```
style to the more compact
```js
export const fn = (...args) => {
  ...body;
};
harden(fn);
```
style. Better to see the `export` where the function is declared.

Reply:
I'm resolving this without changes, since the style I say we're trying to move away from is endemic in our code. We should start retiring it from our code some before changing the docs to explain the new style.